api/src/api/home.js

```diff
@@ -16,4 +17,17 @@ router.get(
   }),
 );
 
+router.get(
+  '/registeredOnly',
```

Comment:
Is this endpoint only for testing?

Reply:
Yes, left this endpoint in there in order to show how to use one of the auth middlewares. We'll remove all of home.js eventually.